src/main/resources/db/changelog/changes/v1.1.0/insert-instances-definition.xml

```diff
@@ -0,0 +1,120 @@
+         much) ¯\_(ツ)_/¯
+         This also makes it easy to keep this idempotent
+     -->
+    <sql>
+      INSERT INTO entity_type_definition (id, derived_table_name, definition)
+      VALUES ('6b08439b-4f8e-4468-8046-ea620f5cfb74', 'drv_instances', '{}') ON CONFLICT (id) DO
+        DO NOTHING;
+    </sql>
+
+    <update tableName="entity_type_definition">
+      <column name="definition">
+        {
+          "id": "6b08439b-4f8e-4468-8046-ea620f5cfb74",
+          "name": "drv_instances",
+          "private": false,
+          "fromClause": "src_inventory_instance",
+          "columns": [
+            {
+              "name": "id",
+              "dataType": {
+                "dataType": "rangedUUIDType"
+              },
+              "visibleByDefault": true
+            },
+            {
+              "name": "instance_hrid",
+              "dataType": {
+                "dataType": "stringType"
+              },
+              "valueGetter": "src_inventory_instance.jsonb ->> 'hrid'",
+              "filterValueGetter": "lower(${tenant_id}_mod_inventory_storage.f_unaccent(src_inventory_instance.jsonb ->> 'hrid'::text))",
+              "valueFunction": "lower(${tenant_id}_mod_inventory_storage.f_unaccent(:value))",
+              "visibleByDefault": false
+            },
+            {
+              "name": "instance_discovery_suppress",
+              "dataType": {
+                "dataType": "booleanType"
+              },
+              "values": [
+                {
+                  "value": "true",
+                  "label": "True"
+                },
+                {
+                  "value": "false",
+                  "label": "False"
+                }
+              ],
+              "valueGetter": "src_inventory_instance.jsonb ->> 'discoverySuppress'",
+              "filterValueGetter": "\"left\"(lower(src_inventory_instance.jsonb ->> 'discoverySuppress'::text), 600)",
+              "valueFunction": "\"left\"(lower(:value), 600)",
+              "visibleByDefault": false
+            },
+            {
+              "name": "instance_title",
+              "dataType": {
+                "dataType": "stringType"
+              },
+              "valueGetter": "src_inventory_instance.jsonb ->> 'title'",
+              "filterValueGetter": "\"left\"(lower(${tenant_id}_mod_inventory_storage.f_unaccent(jsonb ->> 'title'::text)), 600)",
+              "valueFunction": "\"left\"(lower(${tenant_id}_mod_inventory_storage.f_unaccent(:value)), 600)",
+              "visibleByDefault": true
+            },
+            {
+              "name": "instance_cataloged_date",
+              "dataType": {
+                "dataType": "dateType"
+              },
+              "valueGetter": "src_inventory_instance.jsonb ->> 'catalogedDate'",
+              "visibleByDefault": false
+            },
+            {
+              "name": "instance_metadata_created_date",
```

Comment:
There's an index available for this, but it's one that does a type conversion that we already know breaks FQM